documentation/blog/2025-06-05-whats-in-my-goosehints-file/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "What's in my .goosehints file (and why it probably shouldn't be)"
+description: A deep dive into .goosehints vs Memory MCP, and how to optimize your Goose configuration for better performance
+authors:
+    - ian
+---
+
+![blog cover](blog-banner.png)
+
+# What's in my .goosehints file (and why it probably shouldn't be)
+
+As Goose users, we have two main ways to provide persistent context to our AI assistant: the `.goosehints` file and the Memory MCP server. Today, I'll share what's in my `.goosehints` file, why some of it should probably move to Memory MCP, and how you can make that choice.
```

Comment:
should we refer to it as a the memory extension and link it in the beginning? as soon as you refer it